BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1693,6 +1704,8 @@ IEnumerator FlightAnimRoutine()
         }
         IEnumerator BoostRoutine()
         {
+            float Burnrate = boosterMass / boostTime; //define the amount of fuel per unit of time will be burned
```

Comment:
The preferred convention for variable names is camelCase with the first letter being lower case, sometimes preceded by an underscore if it's intended to be private.
Functions, classes and a few exceptional variables (mostly certain static ones) should use camelCase with the first letter being upper case.